drivers/pwm/pwm_shell.c

```diff
@@ -23,7 +23,7 @@ struct args_index {
 
 static const struct args_index args_indx = {
 	.device = 1,
-	.pwm = 2,
+	.channel = 2,
```

Comment:
`struct args_index` has no field `.channel`. This file can no longer be compiled.

Reply:
thanks for spotting this issue, fix https://github.com/zephyrproject-rtos/zephyr/pull/45202